jscomp/test/tagged_template_test.js

```diff
@@ -0,0 +1,62 @@
+// Generated by ReScript, PLEASE EDIT WITH CARE
+'use strict';
+
+var Mt = require("./mt.js");
+var Caml_array = require("../../lib/js/caml_array.js");
+var Tagged_template_libJs = require("./tagged_template_lib.js");
+
+var query = Tagged_template_libJs.sql`SELECT * FROM users WHERE id = 5`;
```

Comment:
hmm the compiler should not inline the values here

Reply:
I added a test case for this

Reply:
Why should the compiler not inline this?